content/2021/_index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "JakartaOne 2021"
+seo_title: "JakartaOne Live - December 7th, 2021 | Jakarta EE Software | Cloud Native"
+headline: "JakartaOne Live Stream 2021"
+date: 2018-04-05T15:50:25-04:00
```

Comment:
What does this date do? Should we use a recent date or an old one like so?

Reply:
The date is used to determine whether an asset should be included in the build, this represents publish date and can be used to schedule content. It doesn't really matter for us in this case